### PR TITLE
Added two new columns (Proposed Total and Claimed). So that Amy and F…

### PR DESCRIPTION
**Pull request by eoliveros**
…inance not having to keep looking at the details to see how much has been claimed.

### CODE REVIEW

**Comment by djpnewton** [comment]
get rid of the detail link column and just make the total and/or claimed field a link too

File: zapd/templates/admin/payments.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
                     <th>Message</th>
                     <th>Amount</th>
                     <th>Status</th>
+		    <th>TransactionID</th>
```
Comment (djpnewton) [inline zapd/templates/admin/payments.html]: whitespace mismatch

File: zapd/templates/admin/payments.html [hunk 2]
```diff
@@ -24,6 +25,7 @@
                         <td>{{payment.message}}</td>
                         <td>{{payment.amount|int2zap}} ZAP</td>
                         <td>{{payment.status}}</td>
+			<td>{{payment.txid}}</td>
```
Comment (djpnewton) [inline zapd/templates/admin/payments.html]: whitespace mismatch

Comment (eoliveros) [reply zapd/templates/admin/payments.html]: another push.... that should do it.

**Pull request merged**